src/main/java/com/xiaomi/infra/pegasus/rpc/async/TableHandler.java

```diff
@@ -238,7 +270,7 @@ void onRpcReply(
         logger.warn(
             "{}: replica server({}) doesn't serve gpid({}), operator({}), try({}), error_code({}), need query meta",
             tableName_,
-            cachedHandle.session.name(),
+            cachedHandle.primarySession.name(),
             operator.get_gpid().toString(),
```

Comment:
Why only `primarySession` log?

Reply:
What others do you want to add here? I think infos about primary session is enough here. For most of user requests, secondaries are transparent to them, because they make backup request disabled.

Reply:
if primary requst failed, we will get the "erro" log, but backup-requst is success. I think we need the primary and secondary request result log